packages/suite/src/components/suite/modals/ReduxModal/UnhideTokenModal.tsx

```diff
@@ -46,7 +58,14 @@ export const UnhideTokenModal = ({ address, onCancel }: UnhideTokenModalProps) =
             icon="warningTriangleLight"
             iconVariant="warning"
             bodyHeading={<Translation id="TR_UNHIDE_TOKEN_TITLE" />}
-            text={<Translation id="TR_UNHIDE_TOKEN_TEXT" />}
+            body={
+                <>
+                    <Translation id="TR_UNHIDE_TOKEN_TEXT" />
+                    <StyledCheckbox isChecked={checked} onClick={() => setChecked(!checked)}>
```

Comment:
Same here, you can also use it without override

Reply:
1310c923f49d49e10de4b4ab03ac698d506e59cc